### ISSUE TITLE
Issue with "Allowing cross-origin use of images and canvas": (Glitch example not working)

### ISSUE THREAD

**Issue by ArthurRSquiD** [mdn/content#5389]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image

#### Specific section or headline?
At the end of the "Receiving and saving the image" section

#### What did you expect to see?
A working example

#### Did you test this? If so, how?
clicked on the links and get redirected to an error page

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/html/cors_enabled_image`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/cors_enabled_image/index.html
* Last commit: https://github.com/mdn/content/commit/1f2db9f0a46b99c254476b5f9445a74b98a6fa30
* Document last modified: 2021-03-27T00:17:29.000Z

</details>

**Comment by rachelandrew** [comment]
@wbamberg on the subject of demos, this one appears to have gone missing. I'm not sure what we want to do with things like this.

**Comment by wbamberg** [comment]
This looks a lot like the examples that we used to use the samples-server for - somewhere the example needs a server-side component. And in this case whoever wrote this example didn't use the samples-server, probably because they didn't know it existed.

We need to find a way to fix this. I've converted the issue about that to a discussion: https://github.com/mdn/content/discussions/5772 and added a link to this issue too.
